src/commands/edit.ts

```diff
@@ -282,22 +282,50 @@ export async function replaceCharacters(
  *
  * @keys `&` (kakoune: normal)
  */
-export function align(
-  _: Context,
-  selections: readonly vscode.Selection[],
+export function align(_: Context, fill: Argument<string> = " ") {
+  return edit((builder, selections) => {
+    const sortedSelections = Selections.sort(Direction.Forward, [...selections]);
+
+    // Group selections by 'column', nth column being nth selections of each line.
```

Comment:
I was a bit confused at first, since "the nth column" often means "a line's nth character" in the context of a text editor. Perhaps `selectionsToAlign` or `selectionGroups` might be more understandable? Otherwise, please emphasize that "column" here is different from the typical meaning of "column" in a text editor in the comment.